MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitSmm.c

```diff
@@ -2,10 +2,12 @@
   This driver will register two callbacks to call fsp's notifies.
```

Comment:
Like other file, please update this to say something about it being the file with Traditional MM specific functionality.

Reply:
Added a description for the file.